code/go/0chain.net/miner/miner/worker.go

```diff
@@ -54,35 +54,12 @@ func TransactionGenerator(c *chain.Chain, workdir string) {
 
 	GenerateClients(c, numClients, workdir)
 
-	viper.SetDefault("development.txn_generation.max_transactions", c.BlockSize())
-	blockSize := viper.GetInt32("development.txn_generation.max_transactions")
-	if blockSize <= 0 {
-		return
-	}
-
 	// validate the maxFee and minFee, maxFee must > minFee, otherwise, will panic
 	if maxFee-minFee <= 0 {
 		logging.Logger.Panic(fmt.Sprintf("development.txn_generation.max_txn_fee must be greater than "+
 			"development.txn_generation.min_txn_fee, max_fee: %v, min_fee: %v", maxFee, minFee))
 	}
 
-	switch {
-	case blockSize <= 10:
```

Comment:
looks like we should calculate `blockSize` as `totalCost/txnCost`
`totalCost` is in config  `    max_block_cost: 10000 #equal to 100ms`
for txnCost it is a little bit more trickier, but looks like we only generate transfer transactions and now we have cost for it in configs `    transfer_cost: 10`